test/tools/jsontests/BlockChainTests.cpp

```diff
@@ -963,12 +964,6 @@ class bcTestFixture {
 		test::BlockchainTestSuite suite;
 		string const& casename = boost::unit_test::framework::current_test_case().p_name;
 
-		if (casename == "bcForgedTest" && test::Options::get().filltests)
-		{
-			suite.copyAllTestsFromFolder(casename);
-			return;
-		}
-
```

Comment:
What happened to these tests that were copied before this PR?

Reply:
renamed with new suffix Copier.json and are prossed in runAllTests